data/json/items/classes/comestible.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "id": "food",
```

Comment:
Setting comestible type and symbol is reasonable, but volume:1 is not a property of an edible object, there are a large number of counterexamples, and as I've told you Re: gun inheritance, it's not ok to spread out item stats across various levels like this.